src/System.Text.Json/tests/JsonTestHelper.cs

```diff
@@ -696,6 +697,7 @@ public static void AssertContentsNotEqual(string expectedValue, ArrayBufferWrite
                     );
 
             // Temporary hack until we can use the same escape algorithm on both sides and make sure we want uppercase hex.
+            // Todo: create new AssertContentsNotEqualAgainJsonNet to avoid calling NormalizeToJsonNetFormat when not necessary.
```

Comment:
Please create tracking issues for TODOs like this. They are easier to track/manage then searching for "todo" in source. Also, increases visibility for others to pick up issues like this.